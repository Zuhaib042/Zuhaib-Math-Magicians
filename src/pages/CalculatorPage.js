import React from 'react';
import Calculator from '../components/Calculator';
import Header from '../components/Header';

export default function CalculatorPage() {
  return (
    <div>
      <Header />
      <div className="calculatorPage">
        <h2 className="text-4xl text-white font-semibold my-12 px-2 sm:ml-6">
          Lets do some Math!
        </h2>
        <Calculator />
      </div>
    </div>
  );
}
