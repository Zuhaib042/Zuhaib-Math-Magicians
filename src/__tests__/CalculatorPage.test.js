import { render, screen } from '@testing-library/react';
import CalculatorPage from '../pages/CalculatorPage';

it('should render the Calculator page correctly', () => {
  render(<CalculatorPage />);
  const calculatorPage = screen.getByText(/lets do some math/i);
  expect(calculatorPage).toMatchSnapshot();
});
