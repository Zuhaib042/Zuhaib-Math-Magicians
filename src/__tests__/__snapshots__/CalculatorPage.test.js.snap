// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Calculator page correctly 1`] = `
<h2
  class="text-4xl text-white font-semibold my-12 px-2 sm:ml-6"
>
  Lets do some Math!
</h2>
`;
